src/prac.js

```diff
@@ -0,0 +1,23 @@
+front() {
```

Comment:
Please let me know if you want feedback on this file - it looks like a scratchpad that was inadvertently included in the PR?